### ISSUE TITLE
Re-fiddle Gratipay

### ISSUE THREAD

**Issue by Gastove** [magit/magit#1901]
This is a weird issue, but:

I went to try and give y'all dollars on Graitpay, and it turns out that their new changes mean that only "teams" can be given money. So, until `magit` is a "team", no new gittips can come in. 

If you have a sec (and are able to), make `magit` into a team! Then the gratipaying can happen :dango:

**Comment by dunn** [comment]
Does the [Flattr page](https://flattr.com/thing/4143008/Magit-A-Git-Porcelain-inside-Emacs) still work?  (I'm not a big fan of Gratipay anyway; [remember this](http://geekfeminism.wikia.com/wiki/Gittip_crisis)?)

**Comment by Gastove** [comment]
Yes, I remember that; not proud to admit this, but I hadn't caught that Gratipay and Gittip were the same fiasco. Damn.

Flattr page works, as does y'all's donate link; will happily use them instead. Will leave it up to team Magit whether or not the Gratipay account gets update, or when to close this.

**Comment by tarsius** [comment]
Thanks for making sure I am aware of this issue. Actually I was already aware but thanks anyway :-)

> If you have a sec (and are able to), make magit into a team!

There are currently ~6 teams which use the "new gratipay" and I assume that this means that the transition isn't as easy as one might wish. I will try to get this up and going before 1th July, when `2.1.0` is released.

But first I have to make sure that my current balance is payed out. I intended to use bitcoin for that but they dropped support for that just in time. :-(

> remember this

Yes, I have read that - I don't remember the details. Someone involved with gratipay said something stupid and then went into defensive mode instead of apologizing. He probably regrets that now, or at least I am willing to give the reason of doubt. We all make mistakes and I think people should be given a second change.

> Does the Flattr page still work?

Yes. Well kind of. I opened that account years ago and so far I have received about as much money through flattr than I used to receive during a single week through gratipay (which also isn't much).
